h2o-automl/src/main/java/ai/h2o/automl/leaderboard/Leaderboard.java

```diff
@@ -25,6 +26,61 @@
  * TODO: make this robust against removal of models from the DKV.
  */
 public class Leaderboard extends Lockable<Leaderboard> {
+
```

Comment:
moved this bloc of static methods on top of the class, as it is common usage in Java. No change in impl.